src/zoid/buttons/component.jsx

```diff
@@ -228,7 +228,8 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                             start:     (url) => {
                                 return new ZalgoPromise((resolve, reject) => {
                                     window.popupBridge.onComplete = (err, result) => {
-                                        return err ? reject(err) : resolve(result.queryItems);
+                                        const queryItems = result && result.queryItems ? result.queryItems : {};
```

Comment:
When testing this locally, what is the value of `result`? I'm wondering what is causing `result.queryItems` to be invalid and require this default value.

Reply:
@gregjopa this is the log that I got:

```
06-23 17:05:55.448 I/chromium( 5672): [INFO:CONSOLE(26178)] "result ---> null", source: http://localhost.paypal.com:8000/sdk/js?client-id=alc_client1&currency=USD&debug=true (26178)
```

![image](https://user-images.githubusercontent.com/5726810/123139208-f6883e80-d423-11eb-8e5a-fa703c098368.png)